### PR TITLE
Sentiment analysis score script added

### PR DESCRIPTION
**Pull request by angelikavilde**

### CODE REVIEW

File: pipeline_reviews/extract.py [hunk 1]
```diff
@@ -27,8 +27,8 @@ def get_reviews_for_game(game_id: int, cursor: str) -> dict:
     cursor = quote_plus(cursor)
 
     try:
-        request = requests.get(f"""https://store.steampowered.com/appreviews/{game_id}
-                    ?json=1&num_per_page=100&cursor={cursor}""", timeout=10)
+        request = requests.get(f"""https://store.steampowered.com/appreviews/
+                {game_id}?json=1&num_per_page=100&language=english&cursor={cursor}""", timeout=10)
```
Comment (gawsalya) [inline pipeline_reviews/extract.py:31]: nice!

File: pipeline_reviews/sentiment.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+"""Sentiment analysis on extracted reviews"""
+
+import pandas as pd
+from pandas import DataFrame
+from pandas.core.series import Series
+import nltk
+from nltk.corpus import stopwords
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+
+from transform import change_column_types #! diff import here check red
```
Comment (gawsalya) [inline pipeline_reviews/sentiment.py:10]: what's the comment for?

**Review by DMcCallion — APPROVED** [review]
Looks good to me!

**Review by gawsalya — APPROVED** [review]
looks good, make sure to write tests for your files!

**Pull request merged**